ble/plf/alif_ble.c

```diff
@@ -71,9 +72,17 @@ static void global_int_stop(void)
 	irq_key = irq_lock();
 }
 
-void platform_reset_request(uint32_t error)
+void platform_reset_request(uint32_t const error)
 {
-	__ASSERT(0, "Platform  reset requested by BLE host stack, err %d", error);
+	if (RESET_MEM_ALLOC_FAIL == error) {
+		__ASSERT(0,
+			 "Running out of heap. Please, increase it. Current %u "
+			 "(ALIF_BLE_HOST_ADDL_PRF_HEAPSIZE)",
+			 CONFIG_ALIF_BLE_HOST_ADDL_PRF_HEAPSIZE);
+		return;
+	}
+
+	__ASSERT(0, "Platform  reset requested by BLE host stack, err %u", error);
```

Comment:
as a question: should there be a while(1) in this function and not return to stack?